descartes_moveit/CMakeLists.txt

```diff
@@ -34,11 +43,14 @@ catkin_package(
     moveit_core
     rosconsole_bridge
     tf
+    eigen_conversions
```

Comment:
We don't usually need to touch the package dependency. Just curious what breaks the build if you don't have it? (Maybe  it means some lower level issues? Like catkin didn't figure out the deep dependency correctly on Windows?)